app/views/application-process.html

```diff
@@ -0,0 +1,94 @@
+{% extends "layouts/main.html" %}
+
+{% set title = "How applications work" %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    href: "/application",
+    text: "Back to application"
+  }) }}
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">The application process for courses starting in September 2023</h1>
+
+      <p class="govuk-body">For courses starting in September 2023, you can start applying a year before they start in October 2022.</p>
+
+      <p class="govuk-body">Most teacher training courses are one year and start in September when the academic year in England starts. Part time courses are usually 2 years.</p>
+      <br>
+        <table class="govuk-table">
+          <thead class="govuk-table__head">
+            <tr class="govuk-table__row">
+              <th scope="col" class="govuk-table__header">Date and time</th>
+              <th scope="col" class="govuk-table__header">What happens</th>
+            </tr>
+          </thead>
+          <tbody class="govuk-table__body">
+            <tr class="govuk-table__row">
+             <td class="govuk-table__cell">4&nbsp;October&nbsp;2022 at 9am</td>
+              <td class="govuk-table__cell">Start <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses" class="govuk-link">finding postgraduate teacher training courses</a> you might want to apply to.</td>
+            </tr>
+            <tr class="govuk-table__row">
+             <td class="govuk-table__cell">11&nbsp;October&nbsp;2022 at 9am</td>
+              <td class="govuk-table__cell">Start applying for courses. You can apply to 4 courses at a time. Courses can fill up quickly, so you should apply as soon as you can.<br><br>You should make sure each course you choose is offered by a different training provider so you can have more chance of success.<br><br>Training providers will have 40 working days to decide to offer you a place on a course. Holiday periods in England over December and March or April are not included in the 40 working days. Each course you apply to will show the date for when providers need to respond by.<br><br>If training providers do not respond in time, your application will become unsuccessful.<br><br>You can apply to 4 courses again if all your applications become unsuccessful.<br><br>You’ll have 10 working days to respond to any offers you receive. If you do not respond, the offer will be automatically declined.</td>
```

Comment:
Suggestion: "be"/"are" instead of "become"?

Also probably good to switch to paragraph tags here instead of line breaks (doesn't matter for the prototype, but just as a reference for the devs)?

```suggestion
              <td class="govuk-table__cell">Start applying for courses. You can apply to 4 courses at a time. Courses can fill up quickly, so you should apply as soon as you can.<br><br>You should make sure each course you choose is offered by a different training provider so you can have more chance of success.<br><br>Training providers will have 40 working days to decide to offer you a place on a course. Holiday periods in England over December and March or April are not included in the 40 working days. Each course you apply to will show the date for when providers need to respond by.<br><br>If training providers do not respond in time, your application will be unsuccessful.<br><br>You can apply to 4 courses again if all your applications are unsuccessful.<br><br>You’ll have 10 working days to respond to any offers you receive. If you do not respond, the offer will be automatically declined.</td>
```